site/src/pages/OrganizationSettingsPage/OrganizationProvisionerJobsPage/OrganizationProvisionerJobsPage.tsx

```diff
@@ -12,17 +12,17 @@ const OrganizationProvisionerJobsPage: FC = () => {
 	const [searchParams, setSearchParams] = useSearchParams();
 	const filter = {
 		status: searchParams.get("status") || "",
+		ids: searchParams.get("ids") || "",
 	};
-	const queryParams = {
-		...filter,
-		limit: 100,
-	} as GetProvisionerJobsParams;
 	const {
 		data: jobs,
 		isLoadingError,
 		refetch,
 	} = useQuery({
-		...provisionerJobs(organization?.id || "", queryParams),
+		...provisionerJobs(organization?.id || "", {
```

Comment:
nit: since you aren't utilizing the truthiness of the possible `""` on the left hand side I'd use `??` instead